pdfarranger/config.py

```diff
@@ -111,6 +111,13 @@ def set_window_size(self, size):
         self.data.set('window', 'width', str(size[0]))
         self.data.set('window', 'height', str(size[1]))
 
+    def set_position(self,position):
+        self.data.set('window','root_x',str(position[0]))
+        self.data.set('window','root_y',str(position[1]))
```

Comment:
Please add a space after each comma here

Reply:
Not sure why the CI doesn't notice this, but I'm quite sure that is the style we use throughout the project.

Reply:
got it

Reply:
done